docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -13,99 +13,190 @@ description: guide for doing a lift and shift migration of airflow to dagster
 
-By default, each job run of your migrated DAGs creates an ephemeral airflow metadatabase scoped to each job run. This means any [Airflow connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) that your DAG depends on will need to be created. To do this, you can provide a `connections` parameter to <PyObject module="dagster_airflow" object="make_dagster_repo_from_airflow_dags_path" />:
+You'll also need to configure the [Airflow connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) that your DAGs depend on. To accomplish this, use the `connections` parameter instead of URI-encoded environment variables.
 
 ```python file=/integrations/airflow/migrate_repo_connections.py
 import os
 
 from airflow.models import Connection
-from dagster_airflow import (
-    make_dagster_repo_from_airflow_dags_path,
-)
+from dagster_airflow import make_dagster_definitions_from_airflow_dags_path
 
-migrated_airflow_repo = make_dagster_repo_from_airflow_dags_path(
-    os.path.join(os.environ["AIRFLOW_HOME"], "dags"),
-    "migrated_airflow_repo",
-    connections=[
-        Connection(conn_id="http_default", conn_type="uri", host="https://google.com")
-    ],
+migrated_airflow_definitions = make_dagster_definitions_from_airflow_dags_path(
+    os.path.abspath("./dags/"),
+    connections=[Connection(conn_id="http_default", conn_type="uri", host="https://google.com")],
 )
 ```
 
-If you're running on Dagster Cloud and any of your connection fields are sensitive, you can securely pass them in using [environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets).
+Iterate as needed until all configuration is correctly ported to your local environment.
 
 ---
 
-## Step 3: Run your migrated DAGs locally
+## Step 6: Move to production
 
-Once you've defined the repository, you can launch Dagit, Dagster's web UI. This allows you to view your newly migrated jobs and launch runs.
+<Note>
+  This step is applicable to Dagster Cloud. If deploying to your infrastructure,
+  refer to the <a href="/deployment">Deployment guides</a> for more info.
+  <br /><br />
+  Additionally, until your Airflow DAGs execute successfully in your local environment, we recommend waiting to move to production.
+</Note>
 
-```bash
-dagster dev -f <your migrated dags>.py
-```
+In this step, you'll set up your project for use with Dagster Cloud.
 
-If your local shell has the correct permissions, you can also test out running the migrated DAGs end-to-end.
+<!-- Once you have your dagster cloud organization setup you can either choose to adapt your existing airflow repository to use the Dagster Cloud CI/CD deployment or copy over your code to the quickstart repository you made. -->
```

Comment:
❓ Pulled this from some commits made after I started.

I'm not sure exactly what this means, specifically the `quickstart repository you made`. What quickstart repo? The one from the Dagster Cloud GS guide?

Reply:
yes the quickstart repo that you made in the cloud GS guide